lib/queuePopulator/LogReader.js

```diff
@@ -378,17 +378,31 @@ class LogReader {
     }
 
     _processLogEntry(batchState, record, entry) {
-        if (entry.value === undefined) {
-            return;
-        }
-        this._extensions.forEach(ext => ext.filter({
-            type: entry.type,
-            bucket: record.db,
-            // removing the v1 metadata prefix if it exists
-            key: transformKey(entry.key),
-            value: entry.value,
-            logReader: this,
-        }));
+        this._extensions.forEach(ext => {
+            /**
+             * object delete entries don't have a value field,
```

Comment:
maybe this comment should be the 'else' block?
i.e. are undefined values only for "delete" events? is there a more explicit way to detect these?

e.g. something like

```
if (entry.type == "delete") {....}                        // Add the missing processing for this case
else if entry.value === undefined) { return; } // Old code, was already here...
```